mono is broken on aarch64-linux

**Describe the bug**
`mono` looks frequently (and currently) broken on aarch64:
https://hydra.nixos.org/job/nixpkgs/trunk/mono.aarch64-linux

Perhaps the build is flaky?

![image](https://user-images.githubusercontent.com/641278/110814489-7dd7ea80-8257-11eb-832d-569a6ee583ca.png)

I see some stuff like this in the log in the failing cases:
```
mkdir -p -- build/deps
/nix/store/j1wvpskk5wkp5ji42in212csp8095k7k-bash-4.4-p23/bin/bash: mono: command not found
make[6]: *** [build/profiles/basic.make:121: build/deps/basic-profile-check.exe] Error 127
*** The runtime 'mono' doesn't appear to be usable.
*** Trying the 'monolite-linux/B886E13F-6276-4BE5-85F8-C6BF5EDBD200' directory.
Microsoft (R) Visual C# Compiler version 2.8.2.62916 (2ad4aabc)
Copyright (C) Microsoft Corporation. All rights reserved.

Microsoft (R) Visual C# Compiler version 2.8.2.62916 (2ad4aabc)
Copyright (C) Microsoft Corporation. All rights reserved.

rm: cannot remove './/build/gensources.exe': No such file or directory
mkdir -p -- ../../class/lib/basic/tmp/
CSC     [basic] Mono.Cecil.dll
```

**Notify maintainers**
<!--
Please @ people who are in the `meta.maintainers` list of the offending package or module.
If in doubt, check `git blame` for whoever last touched something.
-->
@thoughtpolice @obadz

--- a/pkgs/development/compilers/mono/5.nix
+++ b/pkgs/development/compilers/mono/5.nix
@@ -2,7 +2,7 @@
 
 callPackage ./generic.nix ({
   inherit Foundation libobjc;
-  version = "5.20.1.27";
-  sha256 = "15rpwxw642ad1na93k5nj7d2lb24f21kncr924gxr00178a9x0jy";
+  version = "5.20.1.34";
+  sha256 = "12vw5dkhmp1vk9l658pil8jiqirkpdsc5z8dm5mpj595yr6d94fd";
   enableParallelBuilding = true;
 })

